RGTichy/algos1-1c.js

```diff
@@ -0,0 +1,19 @@
+array=[2,3,4,5,6,7];
```

Comment:
you also want to use "var" before defining variables. This scopes them to within the current function and stops bugs.